react/.eslintrc.js

```diff
@@ -9,7 +9,11 @@ module.exports = {
     'overrides': [
         {
             'files': [ '*.ts', '*.tsx' ],
-            extends: [ '@jitsi/eslint-config/typescript' ]
+            extends: [ '@jitsi/eslint-config/typescript' ],
+            parserOptions: {
+                sourceType: 'module',
+                project: [ './tsconfig.web.json', './tsconfig.native.json' ]
```

Comment:
Why do we need this if we are going to require a tsconfig.json file to exist?

Reply:
Because otherwise eslint will complain. Because we override for `'*.ts', '*.tsx'` eslint errors out and says that any *.ts file should be part of a project. So if we have web tsconfig it'll error out on every native file. So we have to options: either create two `eslintrc` files, one for web and one for native (on each one we exclude the files of the other one), or add both tsconfig to projects. This way we can run eslint just once and tsc twice

Reply:
Sounds good, thanks for explaining!